FDApy/representation/functional_data.py

```diff
@@ -0,0 +1,527 @@
+#!/usr/bin/python3.7
+# -*-coding:utf8 -*
+
+"""
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the first line (D212)](https://app.codacy.com/manual/StevenGolovkine/FDApy/pullRequest?prid=5929506)